src/frontend/packages/core/src/shared/shared.module.ts

```diff
@@ -220,6 +223,7 @@ import { UserPermissionDirective } from './user-permission.directive';
     TableCellSidePanelComponent,
     CardProgressOverlayComponent,
     MaxListMessageComponent,
+    TableCellEndpointAddressComponent,
```

Comment:
Add to list of cells for tests

Reply:
I had the code up whilst prodding this, so made the change